src/dndbeyond/content-scripts/character.js

```diff
@@ -631,6 +631,19 @@ function rollSpell(force_display = false) {
             damage_types.push("Arcane Firearm");
         }
 
+        if (character.hasClassFeature("Alchemical Savant") &&
+            character.getSetting("artificer-alchemical-savant", false) &&
+            damages.length) {
+            for (let i = 0; i < damages.length; i++){
+                if (damage_types[i] === "Acid" || damage_types[i] === "Fire" ||
+                    damage_types[i] === "Necrotic" || damage_types[i] === "Poison") {
+                    damages.push(`${character.getAbility("INT").mod < 2 ? 1 : character.getAbility("INT").mod}`);
```

Comment:
`Math.min(1, character.getAbility("INT").mod)` is more readable than the ternary operator here. Same for the healing part

Reply:
Should be Math.max though: That roll must restore hit points or be a damage roll that deals acid, fire, necrotic, or poison damage, and the bonus equals your Intelligence modifier (minimum of +1).

https://www.dndbeyond.com/classes/artificer

Reply:
arggg, yeah, I told you I was tired.. I saw "minimum" and thought Math.min 🤦‍♂️